### PR TITLE
Added Urdu translation

### PR DESCRIPTION
**Pull request by fasalmbt**
pr

Added Urdu translation for the quiz. According to the 2011 census, there were about 60 million Urdu speakers in India. Adding this to main repo will help them in getting literate about COVID-19

### CODE REVIEW

File: src/components/data.json [hunk 1]
```diff
@@ -525,6 +525,530 @@
...
+          "description": "",
+          "hint": "",
+          "answers": [
+            {
+              "option": "جی ہاں میں کرتا ہوں",
+              "title": "اچھا ، یقینی بنائیں کہ آپ اور آپ کے آس پاس کے لوگ ، اچھی سانس کی حفظان صحت پر عمل کریں۔",
+              "description": [
+                "کھانسی یا چھینک آنے پر اپنے منہ اور ناک کو اپنی مڑی ہوئی کہنی یا ٹشو سے ڈھانپیں ، پھر استعمال شدہ ٹشو کو فورا. ہی ضائع کردیں۔ بوندوں سے وائرس پھیلتا ہے۔ اچھی سانس کی حفظان صحت پر عمل پیرا ہوکر آپ اپنے آس پاس کے لوگوں کو سردی ، فلو اور COVID-19 جیسے وائرس سے محفوظ کرسکتے ہیں۔"
+              ],
+             //"youtubeUrl": "https://www.youtube.com/embed/6Ooz1GZsQ70",
```
Comment (bodhish) [inline src/components/data.json]:
```suggestion
```

File: src/components/data.json [hunk 2]
```diff
@@ -525,6 +525,530 @@
...
+              "correctAnswer": true
+            },
+            {
+              "option": "نہیں ، میں نہیں کرتا",
+              "title": "اچھی سانس کی حفظان صحت پر عمل کریں۔",
+              "description": [
+                "یقینی بنائیں کہ آپ اور آپ کے آس پاس کے لوگ اچھی سانس کی حفظان صحت پر عمل پیرا ہیں۔ اس کا مطلب ہے کہ جب آپ کھانسی کھاتے ہو یا چھینک دیتے ہو تو اپنے جھکے ہوئے خم یا ٹشو سے اپنے منہ اور ناک کو ڈھانپیں ، پھر استعمال شدہ ٹشو کو فوری طور پر نمٹا دیں۔",
+                "بوندوں سے وائرس پھیلتا ہے۔ اچھی سانس کی حفظان صحت پر عمل کرتے ہوئے آپ اپنے آس پاس کے لوگوں کو سردی ، فلو اور کورونا وائرس جیسے وائرس سے محفوظ رکھتے ہیں"
+              ],
+              //"youtubeUrl": "https://www.youtube.com/embed/6Ooz1GZsQ70",
```
Comment (bodhish) [inline src/components/data.json]:
```suggestion
```

**Review by bodhish — CHANGES_REQUESTED** [review]

**Comment by fasalmbt** [comment]
@bodhish can you please change that code.

**Review by bodhish — APPROVED** [review]
Thanks the transilation @fasalmbt  :smile:

**Comment by fasalmbt** [comment]
@bodhish May I add some more languages like Punjabi and bengali?

**Pull request merged**